examples/chem-sync-local-flask/local_app/benchling_app/handler.py

```diff
@@ -31,6 +35,10 @@ def handle_webhook(webhook_dict: dict[str, Any]) -> None:
             render_search_canvas(app, webhook.message)
         elif isinstance(webhook.message, CanvasInteractionWebhookV2):
             route_interaction_webhook(app, webhook.message)
+        elif isinstance(webhook.message, UnknownType):
+            logger.debug("Received an unknown message type: %s", webhook.message)
+            canvas_id = webhook_dict["message"]["canvasId"]
+            render_search_canvas_for_created_canvas(app, canvas_id)
```

Comment:
We actually have the `CanvasCreatedWebhookV2Beta` model available in the latest version of the sdk `1.19.0`... I think we should just updated this repo to the latest sdk rather than using the `UnknownType` here (this current code was necessary _before_ version of the sdk that has the new canvas created webhook model was published)

Reply:
Makes sense, I'll update this.